squash/templates/base.html

```diff
@@ -0,0 +1,56 @@
+{% load staticfiles %}
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width,initial-scale=1">
+        <title>{% block title %}SQUASH{% endblock %}</title>
+        <link rel="stylesheet" href="{% static 'css/bootstrap.min.css' %}">
+        <link rel="stylesheet" href="{% static 'css/site.css' %}">
+    </head>
+
+    <body id="{% block body-id %}body{% endblock %}">
+        {% block top-nav-wrapper %}
+        <div class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+            <div class="container">
+                <div class="navbar-header">
+                    <button type="button" class="navbar-toggle"
+                        data-toggle="collapse" data-target=".navbar-collapse">
+                        <span class="sr-only">Toggle navigation</span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                        <span class="icon-bar"></span>
+                    </button>
+                </div>
+                <div class="collapse navbar-collapse">
+                    <ul class="nav navbar-nav">
+                        <li {% if name == 'index' %}class="active"{% endif %}>
+                            <a href="/">SQUASH</a>
+                        </li>
+                        <li {% if name == 'Dashboard' %}class="dashboard"{% endif %}>
+                            <a href="{% url 'page' 'dashboard' %}">Dashboard</a>
+                        </li>
+                    </ul>
+                    <ul class="nav navbar-nav navbar-right">
+                        <li {% if name == 'help' %}class="active"{% endif %}>
+                            <a ref="{% url 'page' 'help' %}">Help</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        {% endblock %}
+        {% block content %}{% endblock %}
+        <footer>
+            <div class="container">
+                <p>&copy; LSST Corporation {% now 'Y' %}</p>
```

Comment:
Proper copyright attribution is AURA/LSST. LSST Corporation is a different legal entity :)